cpp/src/parquet/encoding_test.cc

```diff
@@ -1627,6 +1645,39 @@ TEST_F(TestRleBooleanEncoding, AllNull) {
       /*null_probability*/ 1));
 }
 
+class TestPlainBooleanEncoding : public TestEncodingBase<BooleanType> {};
+
+TEST(TestPlainBooleanArrayEncoding, AdHocRoundTrip) {
+  std::vector<std::shared_ptr<::arrow::Array>> arrays{
```

Comment:
Add a comment pointing to the GH issue number?

Reply:
done